.github/workflows/pull-request.yaml

```diff
@@ -108,3 +108,27 @@ jobs:
           allowed-failures: check-links
           allowed-skips: test-all, publish-web, nightly
           jobs: ${{ toJSON(needs) }}
+
+  build-prqlc:
```

Comment:
My idea was to provide an easy way to try out features added by pull requests.
For example, I believe duckdb builds and uploads duckdb CLI executables with every pull request.

Reply:
OK great, we can try it. 

I do notice our CI queue filling up occasionally, and this would almost double the number of jobs. So if it becomes an issue we could always add a `pr-build-artifacts` label and run it based on that.

Definitely fine to try, I agree that would be a nice thing to have access to.